frontend/src/pages/modelRegistry/screens/RegisteredModels/useRegisteredModelContext.ts

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import useModelArtifactsByVersionId from '~/concepts/modelRegistry/apiHooks/useModelArtifactsByVersionId';
+import useRegisteredModelById from '~/concepts/modelRegistry/apiHooks/useRegisteredModelById';
+import { ModelVersion } from '~/concepts/modelRegistry/types';
+
+export type RegisteredModelContext = {
```

Comment:
I wonder if we should use a word other than `Context` here to name `RegisteredModelContext` and `useRegisteredModelContext`, since it's not a React Context. Maybe `RegisteredModelDeployInfo` or something? I guess we have the preexisting `projectContext` prop in the modals so maybe it's fine. Up to you.

Even if we keep the word Context I think it should have the word Deploy in it to indicate its purpose - `RegisteredModelDeployContext` / `useRegisteredModelDeployContext` maybe?

Reply:
I agree with you and I will go with the `RegisteredModelDeployInfo`. Thanks!